eng/versioning/version_client.txt

```diff
@@ -142,7 +142,7 @@ com.azure:azure-monitor-query;1.1.0;1.2.0-beta.1
 com.azure:azure-monitor-query-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-perf-test-parent;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-quantum-jobs;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-search-documents;11.5.2;11.6.0-beta.5
+com.azure:azure-search-documents;11.5.2;11.6.0-beta.6
```

Comment:
If the stable latest release was `11.5.4` we should also update that on `main` and all places where it's used.

Reply:
Also, would you mind updating the README to mention this version as well?